examples/Console/TestPrometheusExporter.cs

```diff
@@ -49,7 +49,11 @@ internal static object Run(int port, int totalDurationInMins)
             */
             using var meterProvider = Sdk.CreateMeterProviderBuilder()
                 .AddSource("TestMeter")
-                .AddPrometheusExporter(opt => opt.Url = $"http://localhost:{port}/metrics/")
+                .AddPrometheusExporter(opt =>
+                {
+                    opt.StartHttpListener = true;
+                    opt.HttpListenerPrefixes = new string[] { $"http://*:{port}/" };
```

Comment:
nit: there is a message printed to console showing the path where metrics is available, which still has /metrics in it..